.env.example

```diff
@@ -7,4 +7,5 @@ XDEBUG_INSTALL=true
 XDEBUG_START_WITH_REQUEST=yes
 XDEBUG_MODE=debug
 XDEBUG_LOG_LEVEL=0
-MAPBOX_TOKEN=""
```

Comment:
I wouldn't delete environmental variables without checking if they aren't used anywhere in the project. With this change, you will break backend behavior. This modification will make token undefined in `backend/Services/CoordinateFetcher.php`.

Either we need to use one common name, or use two separate variables for backend and frontend.

Reply:
Yup, deleting variables shouldn't be a case here. In Vue we need to use VUE_APP at the beggining of environmental variable, so either @plencka can change hers or we can have to separate ones.

Reply:
I wasn't aware of the vue's requirement.

I think you can change the MAPBOX_TOKEN to STATIC_API_MAPBOX_TOKEN, so it will be known where it's used from name alone. Remember to update the specified .php file while doing so.

Reply:
@plencka please change the name of your MAPBOX_TOKEN to VUE_APP_MAPBOX_TOKEN. There is no reason to have two seperate variables i quess.

Reply:
Are you sure? [Here's nice webpage regarding security and analytics of tokens.](https://docs.mapbox.com/help/troubleshooting/how-to-use-mapbox-securely/)
With separate tokens, we can easily track down issues from increased responses.

Reply:
Ok, so lets leave both. It will be in next commit.
![image](https://user-images.githubusercontent.com/73656796/127660509-9fe29bce-1875-41d5-bdad-7ee030320545.png)